config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -0,0 +1,71 @@
+periodics:
+  - name: soak-tests-capz-windows-2019
+    interval: 1w
+    decorate: true
+    decoration_config:
+      timeout: 8h
```

Comment:
Will this time out after 8 hours?  Is the idea to run it once over a long time period or multiple times against the same cluster that is long lived?

Reply:
My initial thought was to add a loop which will run clusterloader against the same cluster until it times out.
The `ci-entrypoint.sh` script will cleanup the workload cluster after it finishes executing the commands passed as arguments.

Please let me know if you have other ideas. I am open to suggestions

Reply:
Can we merge this with the 8 hour timeout and iterate after? 
If we merge this we can start validation on some of the other PRs in per-tests like https://github.com/kubernetes/perf-tests/pull/2051

Reply:
This was a comment on the periodic job that this initially added and isn't really valid comment here. Let's discuss the best way to do the soak aspect of this at the next sync.